relayer/pkg/chainlink/config/config.go

```diff
@@ -44,70 +41,6 @@ type Config interface {
 	RequestTimeout() time.Duration
 }
 
-var _ Config = (*config)(nil)
-
-type config struct {
-	defaults ConfigSet
-	dbCfg    db.ChainCfg
-	lggr     logger.Logger
-}
-
-func NewConfig(dbCfg db.ChainCfg, lggr logger.Logger) *config {
-	return &config{
-		defaults: DefaultConfigSet,
-		dbCfg:    dbCfg,
-		lggr:     lggr,
-	}
-}
-
-func (c *config) OCR2CachePollPeriod() time.Duration {
```

Comment:
just wondering, where do these get implemented now?

Reply:
The same types that are used to decode the TOML

Reply:
They should be moved in to these repos eventually.